config/initializers/rack_attack.rb

```diff
@@ -0,0 +1,71 @@
+class Rack::Attack 
```

Comment:
Since this is in an initializer, is there a way to _not_ load it unless the ui or api is present?

Reply:
Maybe check `if defined?(Rails::Server)`, [which is a rack server](https://github.com/rails/rails/blob/6-0-stable/railties/lib/rails/commands/server/server_command.rb#L12). like we do [here](https://github.com/ManageIQ/manageiq/blob/89e27ebd846e5fc8d3f9ee05b37e4a0a91f3c2bd/lib/vmdb/initializer.rb#L9) and  [here](https://github.com/ManageIQ/manageiq/blob/89e27ebd846e5fc8d3f9ee05b37e4a0a91f3c2bd/app/models/authenticator/base.rb#L255).

Reply:
I am on board with that.  Works for me.  Probably where we put the `require 'rack-attack'` in as well.

Reply:
Done.